config/src/comments.rs

```diff
@@ -228,7 +228,7 @@ fn comments() -> HashMap<String, String> {
 #ban_window = 10800
 
 #maximum number of peers
-#peer_max_count = 25
+#peer_max_count = 250
```

Comment:
Unrelated - but can we use `const` from code elsewhere to parameterize/templatize these values?
I haven't looked to closely into how painful this would be to do this but its a .rs file so should be possible?